pep/app/pods/components/page/nav/component.ts

```diff
@@ -37,6 +49,25 @@ export default class PageNav extends Component<PageNavArgs> {
         return this.auth.openLoginModal(true);
     }
 
+    /**
+     * Open the about modal dialog
+     *
+     * @returns Promise<Void>
+     * @memberof PageDrawer
+     */
+    @action
+    async openAboutModal() {
```

Comment:
why are you `await`ing the promise in this `openAboutModal`, but not in the page/drawer version? I think we should probably be consistent either way, and if we await it, show the `loadingBar` while its waiting for the response.

I know its not a ton of code right now, but don't love that this logic is duplicated in 2 places, you could just make an `about` service (see my `auth` service for example), which would be a nicer place for that `ServerStatus` interface to live too probably rather than in this component.

Reply:
I do like the idea of having a nice home for that interface, but I feel like creating a service for this is overkill. You'll probably end up with a large abundance of services that only do minimal work and will make it harder to keep a mental model of the project. Id lean towards keeping it like this unless is grows?

Reply:
Perhaps we should have a service that groups all of this stuff together. Like a `MenuService` or something. That way your not creating one off services all the time for such small functionality

Reply:
yeah i totally agree that having separate services for everything is not the best either, that was my one reservation in suggesting that.

Actually, heres a better solution and probably what we should be doing here; since both the page/nav and page/drawer components are children of the application controller, we can move the `openAboutModal` action into the controller, and pass it down as an arg to both components, `<Page::Nav @openAboutModal={{this.openAboutModal}} />`. 

And the application/controller is a bit of a nicer more central place for the `ServerStatus` interface to live too I think.

Reply:
Yeah, having a more generic service for more than just "about" was gonna be my other suggestion before i thought of the controller/arg solution. But id be fine with either.